1.8.1 release notes (#1545)
astronomer/docs: astro/cli/release-notes.md

If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).

## Astro CLI 1.8.1

Release date: November 23, 2022

### Bug fixes

> - Fixed an issue where you could not use `astro deploy` without a `.env` being present in the Astro project

Reviewer: ```suggestion
- Fixed an issue where you could not use `astro deploy` if you did not have an `.env` file in your Astro project.
```
